pkg/didcomm/transport/http/inbound.go

```diff
@@ -21,6 +21,11 @@ import (
 
 var logger = log.New("aries-framework/http")
 
+const (
+	// acceptInboundContentType additional content type to be accepted for inbound messages.
+	acceptInboundContentType = "application/ssi-agent-wire"
```

Comment:
@swcurran @andrewwhitehead when does this old media type go away?

Reply:
Again need to test with the .NET framework, last I checked it only accepts this one.

Reply:
https://github.com/hyperledger/aries-cloudagent-python/issues/1044